template/{% if use_git %}CONTRIBUTING.md{% endif %}.jinja

```diff
@@ -57,19 +57,17 @@ project and everybody has limited time.
 Ready to contribute? Here's how to set up {{ project_name }} for
 local development.
 
-1. Fork the {{ dev_platform_url }}/{{ username }}/{{ project_slug }}
+1. Fork the {{ dev_platform_url }}/{{ username }}/{{ python_package_name }}
```

Comment:
Because we don't always have a project-slug, I created a variable to make it from the project_name if the slug isn't available.